.travis.yml

```diff
@@ -14,12 +14,7 @@ php:
     - hhvm
 
 env:
-    global:
-        - TEST_COMMAND="composer test"
-    matrix:
-        - SYMFONY_VERSION=3.2.*
-        - SYMFONY_VERSION=3.1.*
-        - SYMFONY_VERSION=2.8.*
+    - SYMFONY_VERSION=2.8.* TEST_COMMAND="composer test"
```

Comment:
@Nyholm it is not the same than setting `SYMFONY_VERSION=3.2.*`:

- it runs tests using the unmodified composer.json, ensuring that your composer.json has the necessary requirement (remove the symfony requirement currently, and you won't see any failure on Travis as it will always re-add one)
- it runs tests against the latest stable version of Symfony, without requiring you to update the version every 6 months.

My own rule for the Travis matrix is this one:

- unmodified composer.json for normal jobs (on different PHP version)
- extra jobs for supported LTS branches of Symfony
- an extra job checking lowest deps (and without changing the Symfony requirement, otherwise this one is not validated)
- an extra job changing the minimum-stability to dev, to ensure compatibility with the upcoming dev version (this one may be allowed to fail depending on your policy)

With such config, I only need to change the build matrix when the list of supported LTS versions is changing (every 2 years)